src/gui/base/InfoBanner.ts

```diff
@@ -41,10 +41,11 @@ export class InfoBanner implements Component<InfoBannerAttrs> {
 				},
 			},
 			[
-				m(".flex", [
-					m(".mt-s.mr-s", this.renderIcon(icon, type ?? null)),
-					m(".flex-grow", [
-						m(".mr.pt-s", [m(".small", lang.getMaybeLazy(message))]),
+				m(".mt-s.mr-s", {style: {position: "absolute"}}, this.renderIcon(icon, type ?? null)), // absolute position makes the icon fixed to the top left corner of the banner
+				m("",
+					{style: {margin: "0 0 0 25px"}}, // allow room for the icon
+					[
+						m(".mr.pt-s", [m(".small", {style: {"word-wrap": "break-word"}}, lang.getMaybeLazy(message))]),
```

Comment:
instead of the explicit style prop, try to use the css classes we already defined in the main-styles.ts, like:

https://github.com/tutao/tutanota/blob/f8f26b2cc2c2d05bc3aed4d8893e9ee1eb406b18/src/gui/main-styles.ts#L628-L630

Reply:
Should be good now!